bigtable-client-core-parent/bigtable-hbase-integration-tests-common/pom.xml

```diff
@@ -124,6 +124,12 @@ limitations under the License.
       <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>1.1.3</version>
```

Comment:
nit: should this be a variable like other versions?

Reply:
we hardcode the truth version in every other module. We can extract it to a variable, but thats for another time